packages/sdk-components-react-radix/src/radix-collapsible.ws.ts

```diff
@@ -0,0 +1,106 @@
+import { RadioUncheckedIcon, RadioCheckedIcon } from "@webstudio-is/icons/svg";
+import type {
+  WsComponentMeta,
+  WsComponentPropsMeta,
+} from "@webstudio-is/react-sdk";
+import {
+  propsRadixCollapsible,
+  propsRadixCollapsibleContent,
+  propsRadixCollapsibleTrigger,
+} from "./__generated__/radix-collapsible.props";
+
+export const metaRadixCollapsible: WsComponentMeta = {
+  category: "radix",
+  type: "container",
+  label: "Collapsible",
+  icon: RadioUncheckedIcon,
+  template: [
+    {
+      type: "instance",
+      component: "RadixCollapsible",
+      props: [
+        {
+          name: "open",
+          type: "boolean",
+          value: false,
+          dataSourceRef: {
+            type: "variable",
+            name: "collapsibleOpen",
+          },
+        },
+        {
+          name: "onOpenChange",
+          type: "action",
+          value: [
+            {
+              type: "execute",
+              args: ["open"],
+              code: `collapsibleOpen = open`,
```

Comment:
question: is this variable going to potentially conflict once we have code components added by the user we don't know about?

Reply:
No, this is replaced with data source id